cdap-app-fabric/src/main/java/io/cdap/cdap/config/PreferencesService.java

```diff
@@ -166,58 +179,66 @@ private void deleteConfig(EntityId entityId) {
   /**
    * Get instance level preferences
    */
-  public Map<String, String> getProperties() {
-    return getConfigProperties(new InstanceId(""));
-  }
+  public Map<String, String> getProperties() { return getConfigProperties(new InstanceId("")); }
+
+  public PreferencesDetail getPreferences() { return get(new InstanceId("")); }
+
 
   /**
    * Get namespace level preferences
    */
-  public Map<String, String> getProperties(NamespaceId namespaceId) {
-    return getConfigProperties(namespaceId);
-  }
+  public Map<String, String> getProperties(NamespaceId namespaceId) { return getConfigProperties(namespaceId); }
```

Comment:
it doesn't seem like we should have these methods anymore, with callers just getting them from the PreferencesDetail.

If this would affect a whole bunch of classes, we can also do this in a separate PR.

Reply:
Correct. Added TODO. Clean up in separate PR